ushahidi/platform-client-mzima
# Removing call to v3-endpoint for post-details

Removing an extra call to V3 to get form-info for displaying form-name when displaying a post.
To test:
1. Go to data-view
2. Click on a post
3. The post should display correctly with all form-info present
